docs/content/latest/explore/multi-region-deployments/row-level-geo-partitioning.md

```diff
@@ -22,11 +22,11 @@ showAsideToc: true
   </li>
 </ul>
 
-
 Row-level geo-partitioning allows fine-grained control over pinning data in a user table (at a per-row level) to geographic locations, thereby allowing the data residency to be managed at the database level. Use-cases requiring low latency multi-region deployments, transactional consistency semantics and transparently schema change propagation across the regions would benefit from this feature. 
 
 {{< tip title="" >}}
 Geo-partitioning makes it easy for developers to move data closer to users for:
+
```

Comment:
Git won't let me add comments on unmodified lines.... 
This is for line 37: The second step involves creating partitions in the respective geographic locations using tablespaces

Reply:
Done